### DISCUSSION TITLE
What is the proper Folder Structure and config YAML file?

### DISCUSSION THREAD

**Discussion by 4e576rt8uh9ij9okp** [ultralytics/ultralytics#1043] [Q&A]
Hello,
I'm new at this and I have trouble making a "folder structure" and "custom.yaml" config file to make it work.

I was looking at the YOLOv8 documentation and sadly I couldn't find anything, I might be blind or maybe it's hidden somewhere.

As a example here https://klipper3d.org
you look on the left menu, click "installation and Configuration"
then click "Configuration reference"
and you see all the parameters for the configuration file.

I was looking for something similar so I know what the hell I need to do.

For now my folders look like this:
```
- output
- - images
- - - test
- - - - target
- - - - - image1.jpg
- - - train
- - - - target
- - - - - image22.jpg
- - - val
- - - - target
- - - - - image33.jpg
- - labels
- - - test
- - - - target
- - - - - image1.txt
- - - train
- - - - target
- - - - - image22.txt
- - - val
- - - - target
- - - - - image33.txt
```

Here is my Yaml file
```
path: C:/Users/LinuxIsBetter/Documents/Github/yolotest/output/images

train: train
test: test
val: val

nc: 1
names: ["target"]
```

Also this is the Error I'm getting:
```
FileNotFoundError: train: No labels found in C:\Users\LinuxIsBetter\Documents\Github\yolotest\output\labels\train\target.cache, can not start training. See https://github.com/ultralytics/yolov5/wiki/Train-Custom-Data
```

**Comment by Laughing-q** [comment]
@4e576rt8uh9ij9okp hi v8 shares the same data format like v5, so I think this link [https://github.com/ultralytics/yolov5/wiki/Train-Custom-Data](https://github.com/ultralytics/yolov5/wiki/Train-Custom-Data) could help you.
Also about your folders I think the following structure should work:
```bash
- output
- - images
- - - test
- - - - image1.jpg
- - - train
- - - - image22.jpg
- - - val
- - - - image33.jpg
- - labels
- - - test
- - - - image1.txt
- - - train
- - - - image22.txt
- - - val
- - - - image33.txt
```
